tools/driver/xamarinreflect_main.cpp

```diff
@@ -225,7 +225,8 @@ class ReflectionPrinter : private DeclVisitor<ReflectionPrinter>, private TypeVi
     void handleNominal(NominalTypeDecl *ND, std::string label)
     {
         indents();
-        EnumDecl *ED = isa<EnumDecl>(*ND) ?(EnumDecl *)ND : 0;
+        EnumDecl *ED = isa<EnumDecl>(*ND) ? (EnumDecl *)ND : 0;
+        ProtocolDecl *PD = isa<ProtocolDecl>(*ND) ? (ProtocolDecl *)ND : 0;
         
         _out << "<typedeclaration kind=\"" << label << "\" name=\"" << ND->getName() << "\"";
```

Comment:
Perse code looks good, I'm wondering if we cod do something like the following to make the code easier to read:
```cpp
std::string stringInQuote = R"(This is a "string")";
```
This is from C++11 but AFAIK you should be able to use it and make things easier to read.

Reply:
That's new to me. I looked at the spec and 🤷‍♀? I think part of it for me is that C was my first high level language so the backslashes are like baroque columns. They hold up the ceiling like other columns, but they just have some extra fiddly bits that you can overlook.